.cicd/travis-build.sh

```diff
@@ -5,7 +5,8 @@ set -eo pipefail
 if [[ "$(uname)" == Darwin ]]; then
   ccache -s
   mkdir -p build && cd build && cmake ..
-  make -j$(getconf _NPROCESSORS_ONLN)
+  echo "Building with -j$JOBS"
+  make -j$JOBS
```

Comment:
I do not think we should switch from `$(getconf _NPROCESSORS_ONLN)` or `$(nproc) to a hard-coded `$JOBS` value for Travis CI, like we are with Buildkite. Sooner or later, Travis will upgrade the number of cores their free-tier agents use, and we will never notice and be building at half-speed.   
   
I know we still have to pass in the `$JOBS` variable because `entrypoint.sh` uses it, but we should try something like this in `travis-build.sh`:
```
export JOBS=$(nproc)
``` 
   
After I finish the `db_modes_test` stuff today, I will help you implement this on Linux, as discussed in-person.

Reply:
I have an attempt at this solution by using the ```.travis.yml``` file here. I will watch to see the result of this build.
https://travis-ci.org/EOSIO/eos-vm/jobs/567001379

Reply:
Confirmed working here:
https://travis-ci.org/EOSIO/eos-vm/jobs/567003608#L342